.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ common: &common
         command: git submodule update --init --recursive
     - run:
         name: install dependencies
-        command: pip install --user tox
+        command: pip install --user tox==3.*
```

Comment:
This adds a bit more surface area when updating the tox version. I think that's fine but we should maybe add a comment in `setup.py` to update this as well when we fix the breaking changes. Thoughts?

Reply:
good call. Will add a comment!